core/domain/fs_domain.py

```diff
@@ -341,14 +352,11 @@ def commit(self, filepath, raw_bytes, mimetype=None):
         # required for binary data (i.e. when mimetype is set to
         # 'application/octet-stream').
 
-        if isinstance(raw_bytes, str):
-            raw_bytes = raw_bytes.encode('utf-8')
-        file_content = (
-            raw_bytes if mimetype != 'application/octet-stream' else raw_bytes)
+        file_content = (raw_bytes)
```

Comment:
You don't need to do this, just directly use `raw_bytes`

Reply:
done

Reply:
> > ![image](https://user-images.githubusercontent.com/37413698/151685574-2f6116c3-45da-4715-8a5c-85708b5efd6e.png) ![image](https://user-images.githubusercontent.com/37413698/151685577-894ff20e-6d73-4911-b234-9ecf19db9618.png) Also, I am still getting these errors.
> 
> Is this happening on the current version of the code? If you need help you need to provide more context. Let me know, we can have a call about this.

Yes I have pulled the latest changes from develop.